health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/HypothesisWrapper.js

```
@@ -1,33 +1,428 @@
+const HypothesisWrapper = ({ onSelect, props: customProps }) => {
+    const { t } = useTranslation();
+    const [currentStep, setCurrentStep] = useState(1);
+    const { state } = useMyContext();
+    const [assumptionValues, setAssumptionValues] = useState(customProps?.sessionData?.HYPOTHESIS?.Assumptions?.assumptionValues ||[]);
+    const assumptionsFormValues = customProps?.sessionData?.CAMPAIGN_FORM?.campaignForm //array with key and value 
+    const campaignType =  customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.campaignType?.code
+    const resourceDistributionStrategyCode= customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.distributionStrat?.resourceDistributionStrategyCode
+    const searchParams = new URLSearchParams(window.location.search);
+    const [internalKey, setInternalKey] = useState(() => {
+        const keyParam = searchParams.get("internalKey");
+        return keyParam ? parseInt(keyParam) : 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)